transferbot.sh

```diff
@@ -56,4 +56,4 @@ read_entities() {
 
 read_entities $@ |\
   mangle_data -t "$target_wiki_origin" -p type -p labels -p descriptions -p aliases -p datatype |\
-  wb create-entity --batch --instance "$target_wiki_origin"
+  wb create-entity --max-lag 1 --no-exit-on-error --batch --instance "$target_wiki_origin"
```

Comment:
What makes me think it might be good thing is that _raising_ the value is discouraged when using batch mode:

> If you want that command to be done already, you can go for a more aggressive value and be done with it :warning: should be avoided when you have a long command queue, typically in --batch mode